exosphere-shared/example-apps/docker-service/application.yml

```diff
@@ -0,0 +1,11 @@
+name: docker-app
```

Comment:
Let's call this app `app-with-external-docker-images`. Also change the directory name.